kano/network.py

```diff
@@ -604,28 +687,25 @@ def connect(iface, essid, encrypt='off', seckey=None, wpa_custom_file=None, conn
 
         logger.info("Starting wpa_supplicant for WEP network '%s' to interface %s" % (essid, iface))
         wpafile = '/etc/kano_wpa_connect.conf'
-        associated = False
         if not wpa_conf(essid, seckey, confile=wpafile, wep=True):
             return False
 
         # wpa_supplicant might complain even if it goes ahead doing its job
-        run_cmd("wpa_supplicant -D nl80211,wext -t -d -c%s -i%s -f /var/log/kano_wpa.log -B" % (wpafile, iface))
+        #run_cmd(SUPPLICANT_CMD.format(wpa_custom_file, iface, SUPPLICANT_LOGFILE))
 
+        # TODO: setup a WEP based AP to test the change below
         # Wait for wpa_supplicant to become associated to the AP - key validation.
         # For WEP Open networks it will always proceed, beacuse there is no real authentication,
         # connection will fail during DHCP process. For WEP Shared networks it will fail here if the key is wrong.
-        assoc_timeout = 20  # seconds
-        assoc_start = time.time()
-        while (time.time() - assoc_start) < assoc_timeout:
-            out, _, _ = run_cmd('wpa_cli -p /var/run/wpa_supplicant/ status|grep wpa_state')
-            wpa_state = out.strip('\n')
-            if len(wpa_state) and wpa_state.split('=')[1] == 'COMPLETED':
-                associated = True
-                break
-            time.sleep(0.5)
-
-        if not associated:
-            return False
+        #assoc_timeout = 20  # seconds
+        #assoc_start = time.time()
+        #while (time.time() - assoc_start) < assoc_timeout:
+        #    out, _, _ = run_cmd('wpa_cli -p /var/run/wpa_supplicant/ status|grep wpa_state')
+        #    wpa_state = out.strip('\n')
+        #    if len(wpa_state) and wpa_state.split('=')[1] == 'COMPLETED':
+        #        associated = True
+        #        break
+        #    time.sleep(0.5)
```

Comment:
Given that we aren't using any of this anymore - just delete it. We have it in the git history anyway and just makes things harder to follow.

Reply:
Fixed too